h2o-algos/src/main/java/hex/coxph/CoxPHModel.java

```diff
@@ -361,5 +361,20 @@ protected Futures remove_impl(Futures fs, boolean cascade) {
     return fs;
   }
 
+  @Override
+  public CoxPHMojoWriter getMojo() {
+    return new CoxPHMojoWriter(this);
+  }
+
+  @Override
+  public boolean haveMojo() {
+    final boolean hasInteraction = _parms.interactionSpec() != null;
+    final boolean hasStratification = false;//_parms._stratify_by != null && _parms._stratify_by.length > 0;
```

Comment:
will be fixed?

Reply:
I can remove this. Thanks for noticing. 

The stratification was an issue with the original version of the code, it works now.